### ISSUE TITLE
Continuous Transcribing

### ISSUE THREAD

**Issue by TheGuyDangerous** [TheGuyDangerous/Voicerra#4]
I am currently facing an issue with the microphone not taking continuous input, so i created have created a new page fot continuous transcription untill the error gets resolved.The logic for continuous live transcription is there inside ../Voicerra/lib/Screens/voice_beta.dart but it's in beta , as it's not the most efficient logic.

**Comment by kushcoder12** [comment]
Hey, is your query regarding live speech recognition for an audio stream. (Speech-to-text)

**Comment by TheGuyDangerous** [comment]
Yes , The project uses a dependency called [speech_to_text][https://pub.dev/packages/speech_to_text] . In the troubleshooting section of the dependency, it is stated that "Currently the plugin is designed for short intermittent use, like when expecting a response to a question, or issuing a single voice command. " 

I therefore want a permanent solution to implement continuous speech to text recognition, be it with or without the dependency , either by calling native kotlin/java code or any means. If you can achieve that , it would be a great help :) .

**Comment by TheGuyDangerous** [comment]
> Hey, is your query regarding live speech recognition for an audio stream. (Speech-to-text)

Yes , I'm using a dependency called speech_to_text - https://pub.dev/packages/speech_to_text . In the troubleshooting section of the dependency, it is stated that "Currently the plugin is designed for short intermittent use, like when expecting a response to a question, or issuing a single voice command. "

I am able to achieve Continuous transcription by a workaround (Manually stoping the active speech recognition session and restarting it) but it's too inefficient.

I therefore want a permanent solution to implement continuous speech to text recognition, be it with or without the dependency , either by calling native kotlin/java code or any means. If you can achieve that , it would be a great help :) .
